output/slide-deck.qmd

```diff
@@ -66,7 +66,24 @@ knitr: true
 
 -   <https://unedo365-my.sharepoint.com/:u:/g/personal/dvmorillo_psi_uned_es/EZBXy-Z5U75FsCKM_P0lTTkBzx3eH0IkrczYzmpPiCX31g?e=4ko0WX>
 
-#### \<SECTION\>: <!-- TODO: Technical assistance procedure -->
+#### Technical assistance procedure
+
+-   If you get stuck, copy and paste the error message to the chat (and chances
+    are someone else is also getting the same error!)
+
+<br>
+
+-   The tech help (Kevin) will try to help you resolve it over chat
+
+<br>
+
+-   We will use the raise hand feature (or similar) to see if everyone is ready
```

Comment:
We sort of discussed this, but I think it would be good to pause and let everyone 'finish' or at least signal when they are ready to move on, to make sure they're not too lost with things. Also usually a good time to answer general questions while waiting for their raised hands

Reply:
Ok great, sounds good! Green check is even better!

Reply:
Done in ebf35474652fc7aebcbed66c3312cd862f76137e